### PR TITLE
e2e: set RSS memory usage limit for kubelet/docker

### PR DESCRIPTION
**Pull request by yujuhong**
Part of #21570

### CODE REVIEW

File: test/e2e/kubelet_perf.go [hunk 1]
```diff
@@ -208,16 +205,30 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
...
+				// We set the memory limits generously because the distribution
+				// of the addon pods affect the memory usage on each node.
+				memLimits: resourceUsagePerContainer{
+					"/kubelet":       &containerResourceUsage{MemoryRSSInBytes: 70 * 1024 * 1024},
+					"/docker-daemon": &containerResourceUsage{MemoryRSSInBytes: 85 * 1024 * 1024},
+				},
 			},
+			// TODO(yujuhong): change this test to ~100 pods per node after
+			// --max-pods have been changed.
 			{podsPerNode: 35,
```
Comment (vishh) [inline test/e2e/kubelet_perf.go]: Should we update the test to run `100` pods? Since the limit is lesser than that as of now, we can do it in a separate PR.

Comment (yujuhong) [inline test/e2e/kubelet_perf.go]: Yes, I plan to do it in another PR, after --max-pods is bumped.

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please github issue: #21753

**Comment by yujuhong** [comment]
@k8s-bot e2e test this please. Issue #21940

**Comment by yujuhong** [comment]
k8s-bot test this issue: #21940
